src/main/java/mekanism/generators/common/block/BlockGenerator.java

```diff
@@ -243,8 +243,8 @@ public void randomDisplayTick(IBlockState state, World world, BlockPos pos, Rand
             } else if (tileEntity.facing == EnumFacing.EAST) {
                 switch (type) {
                     case HEAT_GENERATOR:
-                        world.spawnParticle(EnumParticleTypes.SMOKE_NORMAL, (double) (xRandom + iRandom), (double) yRandom + 0.5F, (double) (zRandom - jRandom), 0.0D, 0.0D, 0.0D);
-                        world.spawnParticle(EnumParticleTypes.FLAME, (double) (xRandom + iRandom), (double) yRandom + 0.5F, (double) (zRandom - jRandom), 0.0D, 0.0D, 0.0D);
+                        world.spawnParticle(EnumParticleTypes.SMOKE_NORMAL, xRandom + iRandom, (double) yRandom + 0.5F, zRandom - jRandom, 0.0D, 0.0D, 0.0D);
```

Comment:
why do some maintain the cast here?

Reply:
Unsure I just let IntelliJ remove the ones it said was unneeded, I can look to see if it works properly still without them when I am next editing this.

Reply:
I was able to remove the other ones with it continuing to work fine.